bobtemplates/plone/theme_barceloneta/src/+package.namespace+/+package.name+/theme/index.html

```diff
@@ -1,69 +1,109 @@
 <!doctype html>
-<html>
-  <head>
-    <title>Plone Theme</title>
-    <link rel="shortcut icon" type="image/x-icon"
-          href="barceloneta-favicon.ico" />
-    <link rel="apple-touch-icon"
-          href="barceloneta-apple-touch-icon.png" />
-    <link rel="apple-touch-icon-precomposed" sizes="144x144"
-          href="barceloneta-apple-touch-icon-144x144-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed" sizes="114x114"
-          href="barceloneta-apple-touch-icon-114x114-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed" sizes="72x72"
-          href="barceloneta-apple-touch-icon-72x72-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed" sizes="57x57"
-          href="barceloneta-apple-touch-icon-57x57-precomposed.png" />
-    <link rel="apple-touch-icon-precomposed"
-          href="barceloneta-apple-touch-icon-precomposed.png" />
-  </head>
-  <body>
-    <section id="portal-toolbar">
-    </section>
-    <div class="outer-wrapper">
-      <header id="content-header">
-        <div class="container">
-          <header id="portal-top">
-          </header>
-          <div id="anonymous-actions">
-          </div>
-        </div>
-      </header>
-      <div id="mainnavigation-wrapper">
-        <div id="mainnavigation">
-        </div>
+<html class="h-100">
+
+<head>
+  <title>Plone Theme</title>
+  <link rel="apple-touch-icon" href="++theme++barceloneta/barceloneta-apple-touch-icon.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="144x144"
+    href="++theme++barceloneta/barceloneta-apple-touch-icon-144x144-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="114x114"
+    href="++theme++barceloneta/barceloneta-apple-touch-icon-114x114-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="++theme++barceloneta/barceloneta-apple-touch-icon-72x72-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" sizes="57x57" href="++theme++barceloneta/barceloneta-apple-touch-icon-57x57-precomposed.png" />
+  <link rel="apple-touch-icon-precomposed" href="++theme++barceloneta/barceloneta-apple-touch-icon-precomposed.png" />
```

Comment:
i think we can drop all the apple-touch-icon-precomposed files.
this SO answer recommends to not provide precomposed files but only one higher resolution icon which is scaled down by iOS devices. note: the answer is from 2014 and mentions ios 8 which and a icon size of 180px which all seems a bit outdated.
https://stackoverflow.com/a/21144916/1337474

Reply:
I usually use https://realfavicongenerator.net/ to generate the icon set. It differs from what we have but work really good.